ops/services/app_functions/report_stream_batched_publisher/functions/QueueBatchedReportStreamUploader/lib.ts

```diff
@@ -127,8 +128,11 @@ export async function uploadResult(body) {
     "x-functions-key": REPORT_STREAM_TOKEN,
     "x-api-version": uploaderVersion,
     "content-type": "text/csv",
-    client: "simple_report",
+    client: REPORT_STREAM_CLIENT
   });
+  if(process.env.REPORT_STREAM_TOPIC){
```

Comment:
Is this ReportStream topic going to be set directly in the `ENV` variables for the function application? If so, should we also add it to the config and _var files?

Reply:
The config currently checks and will [throw an error if a env variable is not availabe](https://github.com/CDCgov/prime-simplereport/blob/main/ops/services/app_functions/report_stream_batched_publisher/functions/config.ts#L10), so it's not added there. For the same reason since it's not needed currently it's not added in the _var file as that will be done in the next ticket #4524.

Reply:
This if statement is how we make topic optional, correct? So if the var isn't there, that's fine. And since it's not in the config file, no error is thrown. How will this work when it is enabled in config? We either don't throw an error for that var or we make it non-optional?

Reply:
My thinking is that we won't ever add it to config and just leave it as is since it is an optional environment variable like that. 
When we need to deploy the function we can add the `report_stream_topic` to _var.tf and assign it to the `REPORT_STREAM_TOPIC` in main.tf

Reply:
Gotcha! That sounds fine 👍